Example/iOS Example/WillowConfiguration.swift

```diff
@@ -34,103 +34,108 @@ struct WillowConfiguration {
+            let executionMethod: Logger.ExecutionMethod = .synchronous(lock: NSRecursiveLock())
+        #else
+            // A release build probably logs only important data, omitting verbose debug information.
+            // Also, asynchronous logging can be desirable to not hold up the current thread for each log statement.
+            // Note that the execution method encapsulates the dispatch queue allowing for all logger instances using the same
+            // execution method to share the same queue and be correctly synchronized.
+            let appLogLevels: LogLevel = [.event, .warn, .error]
+            let databaseLogLevels: LogLevel = [.event, .warn, .error]
+            let webServicesLogLevels: LogLevel = [.event, .warn, .error]
+            let executionMethod: Logger.ExecutionMethod = .asynchronous(
+                    queue: DispatchQueue(label: "com.nike.example.logger", qos: .utility)
+                )
+        #endif
+
+        log = createLogger(
+            prefix: "🌳🌳🌳",
             name: "App",
-            modifierLogLevel: [.debug, .info, .event],
-            writers: writers,
+            logLevels: appLogLevels,
             executionMethod: executionMethod
         )
 
-        Database.log = configureLogger(
-            emoji: "🗃🗃🗃",
+        Database.log = createLogger(
+            prefix: "🗃🗃🗃",
             name: "Database",
-            modifierLogLevel: [.sql, .debug, .info, .event],
-            writers: writers,
+            logLevels: databaseLogLevels,
             executionMethod: executionMethod
         )
 
-        WebServices.log = configureLogger(
-            emoji: "📡📡📡",
+        WebServices.log = createLogger(
+            prefix: "📡📡📡",
             name: "WebServices",
-            modifierLogLevel: [.debug, .info, .event],
-            writers: writers,
+            logLevels: webServicesLogLevels,
             executionMethod: executionMethod
         )
     }
 
-    private static func configureLogger(
-        emoji: String,
+    private static func createLogger(
+        prefix: String,
         name: String,
-        modifierLogLevel: LogLevel,
-        writers: [LogLevel: [LogMessageWriter]],
-        executionMethod: LoggerConfiguration.ExecutionMethod)
+        logLevels: LogLevel,
+        executionMethod: Logger.ExecutionMethod)
         -> Logger
     {
-        let prefixModifier = PrefixModifier(emoji: emoji, name: name)
+        let prefixModifier = PrefixModifier(prefix: prefix, name: name)
         let timestampModifier = TimestampModifier()
+        let writers: [LogWriter] = [ConsoleWriter(modifiers: [prefixModifier, timestampModifier]), ServiceWriter()]
 
-        let modifiers: [LogLevel: [LogMessageModifier]] = {
-            let modifiers: [LogMessageModifier] = [prefixModifier, timestampModifier]
+        return Logger(logLevels: logLevels, writers: writers, executionMethod: executionMethod)
```

Comment:
Good question. I thought it should be plural because you pass in a list of levels that you want to log. I gave it the same treatment as other plural arguments like Arrays. That it's a bit mask is an implementation detail. I'm open to discussion on this however....